src/index.js

```diff
@@ -1,13 +1,50 @@
-import _ from 'lodash';
+import './index.css';
 
-function component() {
-  const element = document.createElement('div');
+const taskList = [
+  {
+    description: 'Shopping',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Study ES6 - Webpack',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Complete Awesome Books ES6',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Workout',
+    completed: true,
+    index: 3,
+  },
+];
 
-  // Lodash, currently included via a script, is required for this line to work
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const mainContainer = document.querySelector('.todo-list-container');
+mainContainer.innerHTML = `<div class="row">
+<h1>Today's To Do</h1>
+<i class="fa-solid fa-rotate fa-lg font-awesome-icon"></i>
+</div>
+<div>
+<input placeholder="Add to your list...">
+</div>`;
 
-  return element;
-}
+taskList.forEach((e) => {
```

Comment:
Please define a function that iterates over the tasks array and populates the HTML list item element for each task.
Moreover, On page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the values of the `index`  property for each task. Kindly see the screenshot below for reference.
![vs_pic](https://user-images.githubusercontent.com/35504149/178049470-f0a7ec61-33f0-40f0-8050-57265ddc9d1a.png)

![list](https://user-images.githubusercontent.com/35504149/178049521-8333d324-9749-4d7c-93d4-96b0e5addc72.png)